### PR TITLE
Add Avatar component

### LINKED ISSUE #66: Add User Avatar component
Design #54.

### PR DESCRIPTION
**Pull request by paulocfjunior**
Closes #66.

image

### CODE REVIEW

**Comment by paulocfjunior** [comment]
@matthieuh the changes you suggested have been applied! Now the Avatar component will show itself on a dark or light variant according to the theme, but if you want to override it, you can pass a `variant` prop with the value `dark` or `light`.

avatar component

File: packages/faency/src/components/Avatar.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import React, { useContext, forwardRef } from 'react'
+import { Avatar as AvatarPrimitive, AvatarProps as AvatarPrimitiveProps } from '@modulz/primitives'
+import { ThemeContext } from 'styled-components'
+import useDarkMode from 'use-dark-mode'
```
Comment (matthieuh) [inline packages/faency/src/components/Avatar.tsx]: `use-dark-mode` shouldn't be used inside faency. We are trying to keep components stateless. As a workaround if you want always white or always black why not using hexa codes?

Comment (paulocfjunior) [reply packages/faency/src/components/Avatar.tsx]: sure, makes sense!

**Comment by paulocfjunior** [comment]
@matthieuh it's done! 😄

**Pull request merged**